test/units/testsuite-59.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+set -o pipefail
+
+mkdir -p /run/udev/rules.d/
+
+cat > /run/udev/rules.d/50-testsuite.rules <<EOF
+SUBSYSTEM=="mem", KERNEL=="null", OPTIONS="log_level=debug"
+ACTION=="add", SUBSYSTEM=="mem", KERNEL=="null", IMPORT{program}="/bin/echo -e HOGE=aa\\\\x20\\\\x20\\\\x20bb\nFOO=\\\\x20aaa\\\\x20\n\n\n"
+EOF
+
+udevadm control --reload
+udevadm trigger -c add /dev/null
+
+while : ; do
+    test -f /run/udev/data/c1:3 &&
+        udevadm info /dev/null | grep -q 'E: HOGE=aa\\x20\\x20\\x20bb' &&
+        udevadm info /dev/null | grep -q 'E: FOO=\\x20aaa\\x20' &&
+        break
+
+    sleep .5
+done
+
+echo OK > /testok
+
+exit 0
```

Comment:
I do see that this doesn't fit too well anywhere else, but maybe this could be combined with one of the other tests? Spinning up a VM for this feels wasteful.

Reply:
Test-17 maybe?

Reply:
Ok

Reply:
Merged. PTAL.